app/services/public_transportation/import_service.rb

```diff
@@ -26,6 +26,7 @@ def initialize(gtfs_url, data_provider_id, whitelist_poi = [])
     @whitelist = whitelist_poi.map(&:to_i) if whitelist_poi.present?
     @pois = {}
     @redis = GtfsRedisAdapter.new
+    @import_from_calender_date = (Date.today - (1.day)).strftime("%Y%m%d")
```

Comment:
We do not need `- 1.day` or am I wrong? today at 00:00:00 would be fine I guess.

Reply:
ok